src/stores/artists-store.js

```diff
@@ -0,0 +1,17 @@
+import { autorun, observable } from 'mobx'
+
+class ArtistsStore {
+  @observable artists = []
```

Comment:
Hmm z tego co widzę to nigdzie nie wykorzystujesz `artists`. Do usunięcia :P

Reply:
yup, uzywalem w componentcie testowym, ktory potem wywalilem, zapomnialem o storze